server/runtime/src/main/java/org/infinispan/server/Server.java

```diff
@@ -406,6 +406,7 @@ public ConfigurationInfo getConfiguration() {
       } else {
          loginConfiguration.put("mode", "HTTP");
       }
+      loginConfiguration.put("ready", Boolean.toString(rest.authentication().authenticator().isReadyForHttpChallenge()));
```

Comment:
should not this be present only in case of properties based, on HTTP mode ?

Reply:
that's already handled. The default is to return true (LDAP, Kerberos, OAuth, etc). If you have a properties-based realm then it will be dependent on entries being present